Examples/Framework/src/Framework/Sequencer.cpp

```diff
@@ -361,6 +362,15 @@ int Sequencer::run() {
     }
   }
 
+  // Inform readers that we're going to start from a specific event number
+  if (eventsRange.first > 0) {
```

Comment:
I wonder if it would be less surprising if we explicitly call `skip(0)`

Reply:
Yeah, why not.